MapboxMobileEvents.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |s|
 
   # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.ios.deployment_target = "8.0"
+  s.ios.deployment_target = "9.0"
```

Comment:
I wonder if we should have bump of minimum deployment target in the separate PR/commit?